charts/govuk-sli-collector/templates/cronjob.yaml

```diff
@@ -1,4 +1,6 @@
 {{- $_ := .Values.govukEnvironment | required "govukEnvironment is required" }}
+{{- $imageTagConfig := $.Files.Get (printf "charts/app-config/image-tags/%s/govuk-sli-collector" .Values.govukEnvironment) | fromYaml }}
```

Comment:
I'm not sure if this path will work or how to test it. Any ideas?

Reply:
`helm template`? (see https://github.com/alphagov/govuk-helm-charts/#installing-an-application-chart-without-argo-cd)